pyomo/contrib/doe/doe.py

```diff
@@ -70,7 +70,8 @@ def __init__(
         solver=None,
         prior_FIM=None,
         discretize_model=None,
-        args=None,
+        args={},
```

Comment:
Note that all class instances that use the default value for `args` will *share the same dict*.  This means that if someone changes the args on one instance the others will see the same change.  A better solution is probably to leave the default `None` and then:
```python
if args is None:
    args = {}
self.args = args
```

Reply:
Thank you. Fixed. And I learned something new.